src/components/molecules/VenueInfoEvents/VenueInfoEvents.tsx

```diff
@@ -86,7 +86,7 @@ const VenueInfoEvents: React.FunctionComponent<PropsType> = ({
               eventsNow.length ? "btn-primary" : "btn-secondary"
             } btn-block`}
             // @debt would be nice not to refresh the page
-            onClick={() => (window.location.href = venueInsideUrl(venue.id))}
+            onClick={() => openUrl(venueInsideUrl(venue.id))}
```

Comment:
As above RE: potential clashes with `venueInsideUrl` + function inside `openUrl`

Reply:
apparently I am thinking of `openRoomUrl`. Does that serve the purpose of `venueInsideUrl`?

Reply:
There's no need to use `openRoomUrl` here since we control the generated URL and it'll never not have `http` in it.